test/modules/shadowing/issue-19352-1.chpl

```diff
@@ -0,0 +1,23 @@
+module Library {
+  record rec {
+    proc method() { writeln("Library's rec.method()"); }
+  }
+}
+
+
+module LibraryPlus {
+  import Library;
+  import Library.rec;
+
+  proc rec.method() { writeln("LibraryPlus's rec.method()"); }
+}
+
+// Case 1
+module Program {
+  import Library;
+  proc main() {
+    import LibraryPlus.rec; // does this make LibraryPlus.rec.method "closer" ?
+    var r = new Library.rec();
+    r.method(); // currently: ambiguity error
+  }
+}
```

Comment:
I wouldn't change anything here, but FWIW, I find naming tests after issue numbers to be suboptimal in that the numbers aren't intuitive to me, so don't say anything about what the test is checking without opening up a browser.  I'd prefer to call such a test something about what it does (e.g., ambigTertiaryMethod.chpl) and noting the issue number in a comment instead.  Put another way, when I see a test fail in start_test output, it's nice to have some vague sense of what the test does from its name.